src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarCommercialButtons.jest.tsx

```diff
@@ -1,51 +1,88 @@
 import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
 import { graphql } from "react-relay"
-import { screen, fireEvent } from "@testing-library/react"
+import { screen, fireEvent, waitFor } from "@testing-library/react"
 import { ArtworkSidebarCommercialButtons_Test_Query } from "__generated__/ArtworkSidebarCommercialButtons_Test_Query.graphql"
 import { Toasts, ToastsProvider } from "@artsy/palette"
 import { createMockEnvironment } from "relay-test-utils"
 import { MockBoot } from "DevTools/MockBoot"
-import { ArtworkSidebarCommercialButtonsFragmentContainer } from "Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons"
+import { ArtworkSidebarCommercialButtons } from "Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons"
 import { useAuthDialog } from "Components/AuthDialog"
 import { useRouter } from "System/Router/useRouter"
+import userEvent from "@testing-library/user-event"
+import { useFeatureFlag } from "System/useFeatureFlag"
 
 jest.unmock("react-relay")
 
 jest.mock("System/Router/useRouter")
 
+jest.mock("System/useFeatureFlag", () => {
+  return {
+    useFeatureFlag: jest.fn(),
+  }
+})
+
 jest.mock("Components/AuthDialog/useAuthDialog", () => ({
   useAuthDialog: jest.fn().mockReturnValue({ showAuthDialog: jest.fn() }),
 }))
 
 describe("ArtworkSidebarCommercialButtons", () => {
   let user
+  let partnerOffers
 
-  let mockEnvironment = createMockEnvironment()
+  let mockEnvironment
+  const mockUseFeatureFlag = useFeatureFlag as jest.Mock
 
-  const { renderWithRelay } = setupTestWrapperTL<
+  const { renderWithRelay: baseRenderWithRelay } = setupTestWrapperTL<
     ArtworkSidebarCommercialButtons_Test_Query
   >({
-    Component: ({ artwork }) => (
-      <MockBoot relayEnvironment={mockEnvironment} context={{ user }}>
-        <ToastsProvider>
-          <Toasts />
-          <ArtworkSidebarCommercialButtonsFragmentContainer
-            artwork={artwork!}
-          />
-        </ToastsProvider>
-      </MockBoot>
-    ),
+    Component: ({ artwork, me }) => {
+      return (
+        <MockBoot relayEnvironment={mockEnvironment} context={{ user }}>
+          <ToastsProvider>
+            <Toasts />
+            <ArtworkSidebarCommercialButtons artwork={artwork!} me={me!} />
+          </ToastsProvider>
+        </MockBoot>
+      )
+    },
     query: graphql`
       query ArtworkSidebarCommercialButtons_Test_Query @relay_test_operation {
         artwork(id: "josef-albers-homage-to-the-square-85") {
           ...ArtworkSidebarCommercialButtons_artwork
         }
+        me {
+          ...ArtworkSidebarCommercialButtons_me
+            @arguments(artworkID: "josef-albers-homage-to-the-square-85")
+        }
       }
     `,
   })
 
+  const renderWithRelay: typeof baseRenderWithRelay = (
```

Comment:
I wonder how much benefit this abstraction could bring. 
I think we already have a good tooling for testing we don't/should add anything on top, even if that means the tests being a bit more verbose, these abstractions have 2 big downsides imo:
- creates an extra layer of complexity, whoever comes here will need to read and understand this new abstraction, it moves things away from being straightforward
- creates some unknowns only discovered when you have an upgrade and some tests failing, so you need to come, spend time trying to understand what's going on 

So I'm very in favor of having a bit more of repetition during tests rather than more layers on top of tooling

Reply:
OK, will remove.